src/lib.rs

```diff
@@ -40,21 +40,23 @@ pub fn swap_nonatomic<A, B>(a: A, b: B) -> io::Result<()> where A: AsRef<Path>,
 
 	match fs::rename(b, a) {
 		Ok(_) => (),
-		Err(_) => {
+		Err(e) => {
```

Comment:
Since we're not using `e` directly, `Err(e)` can be matched by `error` or other variable name.

Reply:
:+1: